sonar-batch-protocol/src/main/protobuf/file_source_test_db.proto

```diff
@@ -0,0 +1,59 @@
+/*
+    SonarQube, open source software quality management tool.
+    Copyright (C) 2008-2015 SonarSource
+    mailto:contact AT sonarsource DOT com
+
+    SonarQube is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 3 of the License, or (at your option) any later version.
+
+    SonarQube is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program; if not, write to the Free Software Foundation,
+    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+/*
+Notes
+
+  - "required" fields are not used as recommended by Google to keep forward-compatibility:
+    https://developers.google.com/protocol-buffers/docs/proto#simple
+
+  - the related Java files are not generated during build. Indeed the existing protoc maven
+    plugins require protobuf to be installed on boxes. That means that generated Java files
+    are updated and committed for each change (see src/main/gen-java).
+*/
+
+// structure of db column FILE_SOURCES.TEST_DATA
+
+// Temporarily in sonar-batch-protocol
+
+package org.sonar.server.source.db;
+option optimize_for = SPEED;
+
+message Test {
+  optional string uuid = 1;
+  optional string key = 2;
+  optional string method_name = 3;
+  optional string status = 4;
+  optional string test_message = 5;
+  optional string type = 6;
+  repeated CoverageBlock coverage_block = 7;
+
+  message CoverageBlock {
+    optional string uuid = 1;
+    optional string key = 2;
```

Comment:
ok. the model has already changed in fact. I suggest you review this part thouroughly when database feeding is done